app/Models/Station.php

```diff
@@ -29,22 +29,45 @@ class Station extends Model
     use HasFactory, LogsActivity;
 
     protected $table    = 'train_stations';
-    protected $fillable = ['ibnr', 'wikidata_id', 'rilIdentifier', 'name', 'latitude', 'longitude', 'time_offset', 'shift_time'];
+    protected $fillable = [
+        'ibnr', 'wikidata_id', 'rilIdentifier',
+        'ifopt_a', 'ifopt_b', 'ifopt_c', 'ifopt_d', 'ifopt_e',
+        'name', 'latitude', 'longitude', 'time_offset', 'shift_time'
+    ];
     protected $hidden   = ['created_at', 'updated_at', 'time_offset', 'shift_time'];
```

Comment:
Maybe add the individual ifopt segments to hidden?

Reply:
Why? We don't expose Stations to API consumers, since we are using Resources. So I don't see a reason to use the $hidden property.